shop.html

```diff
@@ -0,0 +1,599 @@
+              </p>
+            </div>
+          </div>
+          <div class="flex justify-end items-center mt-4">
+            <button
+              class="text-travel-orange font-semibold hover:text-orange-600"
+            >
+              More Accessories &rarr;
+            </button>
+          </div>
+        </div>
+        <div>
+          <div class="sm:flex sm:items-baseline sm:justify-between">
+            <h2 class="text-2xl font-bold">Health &amp; Beauty</h2>
+          </div>
+          <div
+            class="mt-6 grid grid-cols-1 gap-y-10 gap-6 md:grid-cols-2 lg:grid-cols-3 lg:gap-x-8"
+          >
+            <div class="group relative">
+              <a href="#">
+                <div
+                  class="h-96 w-full overflow-hidden rounded-lg sm:aspect-h-3 sm:aspect-w-2 group-hover:opacity-75 sm:h-auto"
+                >
+                  <img
+                    src="/assets/images/eyemask.jpg"
+                    alt=""
+                    class="w-full h-96 object-cover object-center"
+                  />
+                </div>
+                <h3 class="mt-4 text-travel-orange text-lg font-semibold">
+                  Eyemask
+                </h3>
+              </a>
+              <p class="mt-1 text-sm">
+                Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum
+              </p>
+            </div>
+            <div class="group relative">
+              <a href="#">
+                <div
+                  class="h-96 w-full overflow-hidden rounded-lg sm:aspect-h-3 sm:aspect-w-2 group-hover:opacity-75 sm:h-auto"
+                >
+                  <img
+                    src="/assets/images/hydrating skincare.jpg"
+                    alt=""
+                    class="w-full h-96 object-cover object-center"
+                  />
+                </div>
+                <h3 class="mt-4 text-travel-orange text-lg font-semibold">
+                  Belkin Wireless Charger
+                </h3>
+              </a>
+              <p class="mt-1 text-sm">
+                Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum
+              </p>
+            </div>
+            <div class="group relative">
+              <a href="#">
+                <div
+                  class="h-96 w-full overflow-hidden rounded-lg sm:aspect-h-3 sm:aspect-w-2 group-hover:opacity-75 sm:h-auto"
+                >
+                  <img
+                    src="/assets/images/kiehls sanitizer and hand cream.jpg"
+                    alt=""
+                    class="w-full h-96 object-cover object-center"
+                  />
+                </div>
+                <h3 class="mt-4 text-travel-orange text-lg font-semibold">
+                  Kiels Hand Sanitiser
+                </h3>
+              </a>
+              <p class="mt-1 text-sm">
+                Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum Lorem ipsum
```

Comment:
Missing alt tags, missing links (maybe link to external page of the product?)

Reply:
done alt tags and also changed product names too as these were duplicated
Just leaving the links as no endpoint as no other way to open the product out, just static placement